.circleci/config.yml

```diff
@@ -265,6 +281,11 @@ workflows:
           context: laa-hmrc-interface-uat
           requires:
             - deploy_main_uat
+      - delete_uat_db:
+          context: laa-hmrc-interface-uat
```

Comment:
is this context the variable that decides what the kubernetes namespace is?

Reply:
That tells circle which context to load the env_vars from 👍